### DISCUSSION TITLE
Pass parameters as attributes for Creative Effect (WebComponent)

### DISCUSSION THREAD

**Discussion by christianarenz** [nolimits4web/swiper#6769] [Q&A]
Is there a way, to configure a <swiper-container> for creative effect just with parameters?
I use the Web Component described [here](https://swiperjs.com/element#install--register-from-npm)

Other basic parameters are working fine.

I tried things like:

```
<swiper-container
    effect="creative"
    creative-effect-prev-translate="[0, 0, -400]"
    creative-effect-next-translate="[100, 0, 0]"
>
```

or …

`creative-effect-prev-translate="0, 0, -400"`

According to the [docs](https://swiperjs.com/element#parameters-as-attributes) this should be possible.

Can someone give a hint?

Thanks

**Comment by nolimits4web** [comment]
It is not possible because these parameters accept Array, but HTML attributes allow to pass only strings
